<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CustomGameDialog</class>
    <widget class="QDialog" name="CustomGameDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>250</width>
                <height>180</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>自定义游戏</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>高度 (9-24):</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="QSpinBox" name="rowsSpinBox">
                    <property name="minimum">
                        <number>9</number>
                    </property>
                    <property name="maximum">
                        <number>24</number>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>宽度 (9-30):</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QSpinBox" name="colsSpinBox">
                    <property name="minimum">
                        <number>9</number>
                    </property>
                    <property name="maximum">
                        <number>30</number>
                    </property>
                </widget>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                    <property name="text">
                        <string>雷数 (10-668):</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="QSpinBox" name="minesSpinBox">
                    <property name="minimum">
                        <number>10</number>
                    </property>
                    <property name="maximum">
                        <number>668</number>
                    </property>
                </widget>
            </item>
            <item row="3" column="0" colspan="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>CustomGameDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>CustomGameDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>